### PR TITLE
Update the LogDir name to include test filter values

### PR DESCRIPTION
**Pull request by lizzha**
Log directory name:
before this change: "2021-03-15-02-03-26-BT53"
after this change: "20210317022848-Functional-0-1-HC48"

### CODE REVIEW

**Comment by squirrelsc** [comment]
@lizzha can we hold this change? We implemented a log parser in V3, which uses the log path to find correct log folder and files. It looks a breaking change to current code, and the dynamic logic is hard to parse now. Any impact, if this change doesn't apply?

**Comment by lizzha** [comment]
> @lizzha can we hold this change? We implemented a log parser in V3, which uses the log path to find correct log folder and files. It looks a breaking change to current code, and the dynamic logic is hard to parse now. Any impact, if this change doesn't apply?

It's for easier to find the corresponding log folders in the ADO pipelines. In lisav3, how do you get the log path?

**Comment by squirrelsc** [comment]
In v3, it starts V2 in a separated folder. So V3 just find the folders with same pattern. But v3 needs to differentiate root log and case log path.

**Comment by lizzha** [comment]
> In v3, it starts V2 in a separated folder. So V3 just find the folders with same pattern. But v3 needs to differentiate root log and case log path.

This change only updates the top level log folder. So you just need to update the name pattern of the top level folder in lisav3.

**Comment by squirrelsc** [comment]
@LiliDeng would you have a quick try on this branch? It's also a chance to test current repo/branch settings. :)

**Comment by squirrelsc** [comment]
> @LiliDeng would you have a quick try on this branch? It's also a chance to test current repo/branch settings. :)

Please test both sequence and parallel runs.

**Comment by LiliDeng** [comment]
@squirrelsc I am testing it.

**Comment by LiliDeng** [comment]
@lizzha Hi, we need update the lisav3 code based on this change, once done will approve this PR.

**Comment by LiliDeng** [comment]
@lizzha Could you please rebase the latest code? Thanks.

**Comment by lizzha** [comment]
> @lizzha Could you please rebase the latest code? Thanks.

rebased

**Review by LiliDeng — APPROVED** [review]
